dangerzone/__init__.py

```diff
@@ -1,6 +1,13 @@
 import os
 import sys
 
+try:
+    from . import vendor  # type: ignore [attr-defined]
+
+    sys.path.insert(0, vendor.__path__[0])
```

Comment:
Nice catch. Actually, I can't use `logging` there, since it's not initialized yet. Still, I can use `print(..., file=sys.stderr)`. Check it out in 3892dec4.

Reply:
Also, check out 68c516f, which logs to `stderr`.

Reply:
Well, we _can_ initialize the `logging` module ourselves, but it's a separation of concerns thing. The `dangerzone/__init__.py` module is part of the Dangerzone library. Libraries tend to log, but not setup the logging infrastructure. The responsibility for this falls to the frontends (e.g., `dangerzone-cli`).

If a library decides to setup logging at import time, then any consumer of this library that wants to setup logging as well will need to build on top of what this library did. This is confusing, so libraries tend not to do that. Does this make more sense?

Reply:
I'm not a fan of logging to `stderr` without going trough a logger at all, because it can pollute the space there.

I'm starting to think we could actually do this `sys.path` changes inside a `vendorized/pymupdf.py` folder living in our code tree, in which case that would be the one we import. This has the following advantages:

1. It is more explicit. We look in the vendorized path, we can use loggers there. 
2. We do not add the `vendor` path for all the libraries that we need, but one by one, potentially configuring them the way we want in their specialized modules.
3. It solves the issue about logging, because by the time this would be imported, it will be configured.

What do you think?

Reply:
We discussed this out-of-band, and landed on adding a `logger.debug()` statement there, without configuring logging ourselves. If the caller has done so, they will see a debug message about vendored packages. If not, they will see nothing. This change can be found in f4073ff013513c2e8b87f57cbba854771819f1b5.